Tests/fixtures/smartphone-42.yml

```diff
@@ -7969,3 +7969,39 @@
     model: X5c Plus
   os_family: Android
   browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 11; M2104K10AC) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 YaApp_Android/21.80.1 YaSearchBrowser/21.80.1 BroPP/1.0 SA/3 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "11"
+    platform: ARM
+  client:
+    type: browser
+    name: Yandex Browser
+    version: 21.80.1
+    engine: Blink
+    engine_version: 92.0.4515.131
+  device:
+    type: smartphone
+    brand: Xiaomi
+    model: Redmi K40 Gaming
```

Comment:
M2104K10AC - Redmi Note 10 Pro 

1) https://www.deviceinfohw.ru/devices/item.php?item=66640
2) https://github.com/search?q=repo%3AKHwang9883%2FMobileModels+M2104K10AC&type=code
3) Google Sertification https://storage.googleapis.com/play_public/supported_devices.html
Redmi	Redmi Note 10 pro	chopin	M2104K10AC
4) https://www.bluetooth.com/qualified-products
| Company Name | Product Name | Model Number | Product Description |
| -- | -- | -- | -- |
| Xiaomi Inc. | Redmi Note 10 Pro | M2104K10AC​ | Mi Mobile Phone |


I think we should keep the old name


https://www.gsmarena.com/xiaomi_redmi_k40_gaming-10880.php
[Models](https://www.gsmarena.com/glossary.php3?term=models)	M2012K10C, M2104K10AC